R/S3_summary.R

```diff
@@ -130,3 +130,17 @@ summarize_tidy <- function(object, test = "t", ...) {
 
   return(tidy_mat)
 }
+
+
+
+#' @export
+summary.lh_robust <- function(object,...){
+  class(object) <- "summary.lh_robust"
+  object
+}
+
+#' @export
+summary.lh <- function(object,...){
+  attr(object, "linearHypothesis")
```

Comment:
1 - shouldn't it be `linear_hypothesis`?
2 - if it should, then this isn't being tested, correct?

Reply:
yes fixing thanks

Reply:
bug remains. pls add tests. also what returns objects of class "lh"

Reply:
Oh I think I misunderstand. is `lh` the class of object linearHypothesis returns?